flixel/math/FlxPoint.hx

```diff
@@ -96,6 +98,20 @@ class FlxPoint implements IFlxPooled
 	}
 	
 	/**
+	 * Set the coordinates of this point (inline).
+	 * 
+	 * @param	X	The X-coordinate of the point in space.
+	 * @param	Y	The Y-coordinate of the point in space.
+	 * @return	This point.
+	 */
+	public inline function setInline(X:Float, Y:Float):FlxPoint
```

Comment:
It seems like the real issue here is the variables having setters for `FlxCallbackPoint`. This is just a really nasty workaround for that...